cli/commands/kms.go

```diff
@@ -143,13 +143,17 @@ func privateKeyHandler(ctx *cli.Context) error {
 	return nil
 }
 
-func publicKeyHandler(ctx *cli.Context) error {
+func addUserToKeyHandler(ctx *cli.Context) error {
 	c, err := getClient(ctx)
```

Comment:
@adrianosela It is down here github is tripping. The logic is so similar that when I removed the get private key all the function names shifted. It is trippy indeed.